data/categories.txt

```diff
@@ -4881,199 +4881,199 @@ zh-Hans:献血
 zh-Hant:獻血
 
 healthcare-optometrist
-en:Optometrist
-ar:طبيب العيون
-be:Акуліст
-bg:Оптометрист
-cs:Optometrista
-da:Optometrist
-de:Augenoptiker
-el:Οπτομέτρης
-es:Optometrista
-et:Optometrist
-eu:Optometrista
-fa:اپتومتریست
-fi:Optometristi
-fr:Optométriste
-he:אופטומטריסט
-hu:Optometrista
-id:Ahli kacamata
-it:Optometrista
-ja:オプトメトリスト
-ko:검안사
-mr:ऑप्टोमेट्रिस्ट
-nb:Optometrist
-nl:Optometrist
-pl:Optometrysta
-pt:Optometrista
-pt-BR:Optometrista
-ro:Optometrist
-ru:Оптометрист
-sk:Optometrista
-sv:Optometrist
-sw:Daktari wa macho
-th:นักทัศนมาตร
-tr:Optometrist
-uk:Оптометрист
-vi:bác sĩ nhãn khoa
-zh-Hans:验光师
-zh-Hant:驗光師
+en:Optometry|Optometrist
+ar:قياس البصر|طبيب العيون
+be:Оптаметрыя|Акуліст
+bg:Оптометрия|Оптометрист
+cs:Optometrie|Optometrista
+da:Optometri|Optometrist
+de:Optometrie|
+el:Οπτομετρία|Οπτομέτρης
+es:Optometría|Optometrista
+et:Optomeetria|Optometrist
+eu:Optometria|Optometrista
+fa:بینایی سنجی|اپتومتریست
+fi:Optometria|Optometristi
+fr:Optométrie|Optométriste
+he:אופטומטריה|אופטומטריסט
+hu:Optometria|Optometrista
+id:Optometri|Ahli kacamata
+it:Optometria|Optometrista
+ja:オプトメトリー|オプトメトリスト
+ko:검안|검안사
+mr:ऑप्टोमेट्री|ऑप्टोमेट्रिस्ट
+nb:Optometri|Optometrist
+nl:Optometrie|Optometrist
+pl:Optometria|Optometrysta
+pt:Optometria|Optometrista
+pt-BR:Optometria|Optometrista
+ro:Optometrie|Optometrist
+ru:Оптометрия|Оптометрист
+sk:Optometria|Optometrista
+sv:Optometri|Optometrist
+sw:Optometry|Daktari wa macho
+th:ทัศนมาตรศาสตร์|นักทัศนมาตร
+tr:Optometri|Optometrist
+uk:Оптометрія|Оптометрист
+vi:Khúc xạ nhãn khoa|bác sĩ nhãn khoa
+zh-Hans:验光配镜|验光师
+zh-Hant:驗光|驗光師
 
 healthcare-podiatrist
-en:Podiatrist
-ar:طبيب الأرجل
-be:Артапед
-bg:Подиатрист
-cs:Podiatr
-da:Fodterapeut
-de:Podologe
-el:Ποδίατρος
-es:Podólogo
+en:Podiatry|Podiatrist
+ar:علاج الأرجل|طبيب الأرجل
+be:Подологія|Артапед
+bg:Подиология|Подиатрист
+cs:Podiatrie|Podiatr
+da:Fodterapi|Fodterapeut
+de:Podologie|Podologe|Podologin
+el:Ποδιατρική|Ποδίατρος
+es:Podología|Podólogo
 et:Jalaravi
-eu:Podologoa
-fa:متخصص پا
-fi:Jalkaterapeutti
-fr:Podologue
-he:פודיאטר
-hu:Podiatrist
-id:Ahli penyakit kaki
-it:Podologo
-ja:足病医
-ko:발 전문의
-mr:पोडियाट्रिस्ट
-nb:Fotterapeut
-nl:Podotherapeut
-pl:Podolog
-pt:Podologista
-pt-BR:Podólogo
-ro:Podiatru
-ru:Ортопед
-sk:Podológ
-sv:Fotvårdare
-sw:Daktari wa miguu
-th:แพทย์โรคเท้า
-tr:Podiatrist
-uk:Ортопед
-vi:Bác sĩ chân
-zh-Hans:足科医生
-zh-Hant:足病醫生
+eu:Podologia|Podologoa
+fa:پزشکی پا|متخصص پا
+fi:Jalkaterapia|Jalkaterapeutti
+fr:Podologie|Podologue
+he:פודיאטריה|פודיאטר
+hu:Podiatria|Podiatrist
+id:Podiatri|Ahli penyakit kaki
+it:Podologia|Podologo
+ja:足病学|足病医
+ko:발병학|발 전문의
+mr:पोडियाट्री|पोडियाट्रिस्ट
+nb:Fotterapi|Fotterapeut
+nl:Podologie|Podotherapeut
+pl:Podiatria|Podolog
+pt:Podiatria|Podologista
+pt-BR:Podiatria|Podólogo
```

Comment:
pt:Podiatria|Podiatra
pt-BR:Podiatria|Podiatra

Reply:
Thanks changed in https://github.com/organicmaps/organicmaps/pull/4053/commits/dad75fe91b511fd26108fab402f7fc0a38781fe2